lib/hz2600/Kazoo/Api/Entity/Search.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Kazoo\Api\Entity;
+
+/*
+TODO: Add support for accounts/{account_id}/search
+*/
+
+class Search extends AbstractEntity
+{
+    public function search($type, $value, $view)
```

Comment:
Can you please put opening curly brace on the same line as the function definition to follow the coding style of the project? 
EG: 
function name() {
}

Reply:
Done